Source/Runtime/Measurement Server.lvproj

```diff
@@ -132,6 +132,7 @@
 			<Item Name="niFgen_64.dll" Type="Document" URL="niFgen_64.dll">
 				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
 			</Item>
+			<Item Name="Initialize and Close Session Behavior.ctl" Type="VI" URL="../MeasurementLink Session Management/Session Reservation/Initialize and Close Session Behavior.ctl"/>
```

Comment:
Strange. This is in Jonathan's PR, which hasn't even been submitted yet.

Reply:
Weird. I think I'll clean things out and kind of start over on this.